src/@types/git.d.ts

```diff
@@ -196,6 +202,8 @@ export interface Repository {
 	getBranches(query: BranchQuery): Promise<Ref[]>;
 	setBranchUpstream(name: string, upstream: string): Promise<void>;
 
+	getRefs(query: RefQuery, cancellationToken?: CancellationToken): Promise<Ref[]>;
```

Comment:
I would copy the whole file `git.d.ts` file as it contains an updated signature for the `getBranches()` method since I added support for cancellation tokens.